HomeWork31/Task2.py

```diff
@@ -0,0 +1,27 @@
+import requests
+
+
+def geocoding(city):
+    geoconing_api = (f"https://geocoding-api.open-meteo.com/v1/search?name={city}&count=1&language=en&format=json")
+    result = requests.get(geoconing_api)
+    data = result.json()
+    latitude = data.get("results")[0].get("latitude")
+    longitude = data.get("results")[0].get("longitude")
+    return latitude, longitude
+
+
+def forecast(location):
+    FORECAST_API = (f"https://api.open-meteo.com/v1/forecast?latitude={location[0]}&longitude={location[1]}&hourly=temperature_2m&current_weather=true&forecast_days=1")
+    response = requests.get(FORECAST_API)
```

Comment:
nit: better to use `params`
```suggestion
    FORECAST_API = "https://api.open-meteo.com/v1/forecast"
    response = requests.get(FORECAST_API, params={
        "latitude": location[0],
        ...
    })
```